src/encoded/tests/data/inserts/expected_publication_data_batch_download.tsv

```diff
@@ -0,0 +1,6 @@
+"http://localhost/metadata/?type=PublicationData&%40id=%2Fpublication-data%2FENCSR727WCB%2F"
+http://localhost/files/ENCFF002ENO/@@download/ENCFF002ENO.tsv
+http://localhost/files/ENCFF002ENS/@@download/ENCFF002ENS.tsv
+http://localhost/files/ENCFF037HIC/@@download/ENCFF037HIC.hic
+http://localhost/files/ENCFF002COS/@@download/ENCFF002COS.bed.gz
+http://localhost/files/ENCFF003COS/@@download/ENCFF003COS.bigBed
```

Comment:
missing newline, not worth fixing by itself

Reply:
Probably just how I'm writing it from test.